WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -86,10 +89,15 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 
 				foreach (var finishedCoinJoin in finishedCoinJoins)
 				{
-					var removed = trackedWallets.Remove(finishedCoinJoin.Wallet.WalletName);
+					var walletToRemove = finishedCoinJoin.Wallet;
+					var removed = trackedWallets.Remove(walletToRemove.WalletName);
 					if (!removed)
 					{
-						Logger.LogWarning($"Wallet: `{finishedCoinJoin.Wallet.WalletName}` was not removed from tracked wallet list. Will retry in a few seconds.");
+						Logger.LogWarning($"Wallet: `{walletToRemove.WalletName}` was not removed from tracked wallet list. Will retry in a few seconds.");
```

Comment:
I believe we write it typically in this way:

```suggestion
						Logger.LogWarning($"Wallet '{walletToRemove.WalletName}' was not removed from tracked wallet list. Will retry in a few seconds.");
```

I don't think we use backticks really.

Reply:
I found more places using backticks, so I'll leave this as it is for now.